src/process/loader-shell-runner.ts

```diff
@@ -71,8 +73,20 @@ export class LoaderShellRunner implements IShellRunner {
     this.tty.showCursor();
   }
 
+  /**
+   * In case of an exception you probably want to force stop the loading animation. Oftherwise the parallel
+   * running commands would still keep the loader spinning which messes with the error text display.
+   * This just ignores the commands running in the background and kills off the animation right away.
+   */
+  private forceStopLoading() {
+    this.runningCommandsCount = 0;
+    this.stopLoading();
+  }
+
   private stopLoading() {
-    this.runningCommandsCount--;
+    if (this.runningCommandsCount > 0) {
```

Comment:
I assume this is just defensive coding, we don't expect that there are legitimate situations in which we decrease runningCommandsCount below zero, right -> except when we do a forceStopLoading, right?

Reply:
a comment in the code with that as explanation might be helpful

Reply:
add a comment.